### PR TITLE
 add span element required in label of inputs components

### PR DESCRIPTION
**Pull request by Gabojcb**

### CODE REVIEW

File: project/modules/form/ts/input.tsx
Comment (jircdev) [inline project/modules/form/ts/input.tsx:1]: Please,  renive the blank lines inside the return statement and all console.log.

File: project/modules/form/scss/input/tag.scss
Comment (jircdev) [inline project/modules/form/scss/input/tag.scss:1]: It is not a good practice to put texts as capitalize, please remove this line.

File: project/modules/form/scss/input/input.scss
Comment (jircdev) [inline project/modules/form/scss/input/input.scss:1]: The width and height properties on lines 69 and 70 can probably be located where the selector starts.

**Review by jircdev — APPROVED** [review]

File: project/modules/form/ts/input.tsx
Comment (Gabojcb) [reply project/modules/form/ts/input.tsx:1]: Thansk, i now correct thats errors

**Review by jircdev — APPROVED** [review]

File: project/modules/form/scss/input/tag.scss
Comment (Gabojcb) [reply project/modules/form/scss/input/tag.scss:1]: I see, i ajustment styles

**Review by jircdev — APPROVED** [review]

**Pull request merged**